src/docs/product/performance/trace-view.mdx

```diff
@@ -12,36 +12,36 @@ If you’re interested in being an Early Adopter, you can turn your organization
 
 </Note>
 
-From Performance, Issues, and Discover, you can drill down into a single [trace](/product/performance/distributed-tracing) and traverse all of the transactions in that trace. This will accelerate your ability to debug slow services, identify related errors, and root out other bottlenecks.
+With this [trace](/product/performance/distributed-tracing) view, you can drill down into the details of a single trace and traverse every transaction in that trace. This will accelerate your ability to debug slow services, identify related errors, and root out other bottlenecks. There are multiple entry points to get to the trace view.
```

Comment:
```suggestion
With the [trace](/product/performance/distributed-tracing) view, you can drill down into the details of a single trace and traverse every transaction in that trace. This will accelerate your ability to debug slow services, identify related errors, and root out other bottlenecks. There are multiple entry points to get to the trace view.
```